cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v2.5
+
+- added a filter for dimensions when user wants to filter by key pair value like Service=AmazonEC2
+
+## v2.4
```

Comment:
What is the reason to update version twice this PR?

Reply:
There no reason, the criterion was taken from the ambiguity that exists in the 2 tickets (HCLIND-2 & HCLIND-59). Either make them an update itself or separately.

Reply:
i left only one update version, thank you